scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/core/JobEmailNotificationTest.java

```diff
@@ -107,6 +107,9 @@ public void setUp() throws KeyException {
     public void testIncompleteFrom() throws Exception {
         setSenderAddress(INCOMPLETE_EMAIL);
         InternalJob job = createJob(USER_EMAIL);
+        Map<String, String> genericInfo = job.getGenericInformation();
+        genericInfo.put("NOTIFICATION_EVENTS", "Job running to finished");
+        job.setGenericInformation(genericInfo);
```

Comment:
is it necessary ? as the local variable genericInfo is already the genericInformation of the job